deploy/osd-logging/01-namespace.yaml

```diff
@@ -3,7 +3,9 @@ kind: Namespace
 metadata:
   name: openshift-logging
   annotations:
-    openshift.io/node-selector: "" 
+    openshift.io/node-selector: ""
   labels:
+    managed.openshift.io/service-lb-quota-exempt: "true"
```

Comment:
```suggestion
```

Reply:
This is following the pattern of other namespaces having this exemption as well.  I followed that pattern here.